titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/event/JobEventFactory.java

```diff
@@ -33,11 +35,14 @@
  */
 public class JobEventFactory implements ReconcileEventFactory<JobManagerReconcilerEvent> {
 
+    public static final CallMetadata UNDEFINED_CALL_METADATA = null;
```

Comment:
I would define this as an empty `CallMetadata` instead of null to remove any type of NPE situation.